style.css

```diff
@@ -282,6 +283,74 @@ body {
     font-weight: 500;
     font-size: 15px;
   }
+
+  /* Contact Form */
+
+  footer {
+    padding-top: 114px;
+  }
+
+  #contact-form {
+    background-image: url(/images/contact-form.png);
+    background-repeat: no-repeat;
+    background-size: 100%;
+    background-color: #6070ff;
+    border-top-left-radius: 100px;
+  }
```

Comment:
- [ ] The form style is excellent, but on some mobile screens, the background image gets overlapped with the white text making it hard to read. I suggest you shrink the background image a little so the white semicircle isn't overlapped by the text.
 -Your design:
<img width="478" alt="image" src="https://user-images.githubusercontent.com/61919617/192517731-4692873f-2809-487c-9796-18014e77cd70.png">

-Figma design:
<img width="412" alt="image" src="https://user-images.githubusercontent.com/61919617/192517776-ce8df0aa-b8a6-4bfe-917b-2a32f173080b.png">